frontend/src/main/components/Commons/CommonsOverview.js

```diff
@@ -1,14 +1,29 @@
 import React from "react";
 import { Card } from "react-bootstrap";
+import moment from "moment";
+
+function getStartDate(timestamp) {
+    return moment(timestamp).format('YYYY-MM-DD');
```

Comment:
Please do this with native JS functions, or with a more modern JS library such as date-fns or luxon.  See: 

https://momentjs.com/docs/#/-project-status/

And: 

* [You Probably Don't Need Moment.js Anymore](https://dockyard.com/blog/2020/02/14/you-probably-don-t-need-moment-js-anymore)
 * [You don't (may not) need Moment.js](https://github.com/you-dont-need/You-Dont-Need-Momentjs/blob/master/README.md)
 * [Why you shouldn't use Moment.js...](https://inventi.studio/en/blog/why-you-shouldnt-use-moment-js)
 * [4 alternatives to moment.js for internationalizing dates](https://blog.logrocket.com/4-alternatives-to-moment-js-for-internationalizing-dates/)

Reply:
@isaiahgarcia I have already created a pure / vanilla JavaScript function to convert timestamps into the desired date format. Feel free to use it. The relevant code is currently awaiting review in [pull request #62](https://github.com/ucsb-cs156-w22/team04-w22-5pm-HappyCows/pull/62).